src/python/genny/genny_auto_tasks.py

```diff
@@ -0,0 +1,74 @@
+import os
+import re
+import sys
+import subprocess
+import textwrap
+
+from shrub.config import Configuration
+from shrub.command import CommandDefinition
+from shrub.variant import TaskSpec
+
+def to_snake_case(str):
+	"""
+	Converts str to snake_case, useful for generating test id's
+	From: https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case
+	:return: snake_case version of str.
+	"""
+	s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', str)
+	return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
+
+def modified_workload_files():
+	"""
+	Returns a list of filenames for workloads that have been modified according to git, relative to origin/master.
+	:return: a list of filenames in the format subdirectory/Task.yml
+	"""
+	out = subprocess.check_output('git diff --name-only `git merge-base HEAD origin/master` -- ../workloads/', shell=True)
```

Comment:
The backticks thing is a legacy thing - instead modern shell recommends using `$()`, so this would become

```
'git diff --name-only $(git merge-base HEAD origin/master) -- ../workloads/'
```

Reply:
Good to know, fixed this.